### PR TITLE
Run a short hammer on Travis

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

**Comment by Martin2112** [comment]
Seems like a good idea, as long as it completes quickly.

File: trillian/integration/ct_hammer/main.go [hunk 1]
```diff
@@ -179,15 +179,17 @@ func main() {
...
 		if e.err != nil {
 			errCount++
-			glog.Errorf("%s: %v", e.prefix, e.err)
+			glog.Errorf("  %s: failed with %v", e.prefix, e.err)
 		}
 	}
 	if errCount > 0 {
 		glog.Exitf("non-zero error count (%d), exiting", errCount)
 	}
+	glog.Info("  no errors; done")
```
Comment (Martin2112) [inline trillian/integration/ct_hammer/main.go]: intentional spaces?

**Comment by daviddrysdale** [comment]
Seems to add around 60-70s to a normal run; not sure about the --race run yet

File: trillian/integration/ct_hammer/main.go (hunk 1 above)
Comment (daviddrysdale) [inline trillian/integration/ct_hammer/main.go]: Yeah -- you get an output like either
```
completed tests on all 3 logs:
  no errors; done
```
or
```
completed tests on all 3 logs:
  athos: failed with some error
  porthos: failed with some-other-error
```

**Comment by daviddrysdale** [comment]
maybe ~+3m for `--race`

**Comment by Martin2112** [comment]
Probably OK for the moment in this repo then.

**Pull request merged**